src/Featurizers/StringFeaturizer.h

```diff
@@ -16,61 +16,110 @@ namespace Featurizers {
 ///  \brief         Converts input into strings.
 ///
 template <typename T>
-class StringTransformer : public Components::InferenceOnlyTransformerImpl<T, std::string> {
+class StringTransformer : public StandardTransformer<T, std::string> {
```

Comment:
This is more of an academic question, but when I was starting this change I left it as an `InferenceOnlyTransofmrer` and just overrode the methods that needed changing. Just curious what the pros/cons are to that approach vs this one that changes it to a `StandardTransformer`?